vendor/yosensi/yo-analog.yaml

```diff
@@ -0,0 +1,87 @@
+name: YO Analog - analog signal monitoring, voltage input (0–10 V) or current input (4–20 MA) | LoRa EU868, US915, AU915, AND AS923
+description: YO Analog makes it possible to monitor devices and processes in automation. The device has six configurable measurement inputs, each of which can operate in one of two analogue modes voltage input (0–10 V) or current input (4–20 mA).
```

Comment:
try to reduce description, and only should be one line

Reply:
What is character limit for the description?

Reply:
There is no limit, but for example in the name of the device it is mention `voltage input` and `current input`, also in description, you can take it out from one of those